components/quickbooks/actions/update-customer/update-customer.mjs

```diff
@@ -1,380 +1,441 @@
-// legacy_hash_id: a_0Mi7vm
-import { axios } from "@pipedream/platform";
+import quickbooks from "../../quickbooks.app.mjs";
+import { ConfigurationError } from "@pipedream/platform";
 
 export default {
   key: "quickbooks-update-customer",
   name: "Update Customer",
   description: "Updates a customer.",
-  version: "0.1.1",
+  version: "0.1.2",
   type: "action",
   props: {
-    quickbooks: {
-      type: "app",
-      app: "quickbooks",
-    },
-    display_name: {
+    quickbooks,
+    displayName: {
+      label: "Display Name",
       type: "string",
       description: "The name of the person or organization as displayed. Must be unique across all Customer, Vendor, and Employee objects. Cannot be removed with sparse update. If not supplied, the system generates DisplayName by concatenating customer name components supplied in the request from the following list: `Title`, `GivenName`, `MiddleName`, `FamilyName`, and `Suffix`.",
       optional: true,
     },
     title: {
+      label: "Title",
       type: "string",
       description: "Title of the person. This tag supports i18n, all locales. The `DisplayName` attribute or at least one of `Title`, `GivenName`, `MiddleName`, `FamilyName`, `Suffix`, or `FullyQualifiedName` attributes are required during update.",
       optional: true,
     },
-    given_name: {
+    givenName: {
+      label: "Given Name",
       type: "string",
       description: "Given name or first name of a person. The `DisplayName` attribute or at least one of `Title`, `GivenName`, `MiddleName`, `FamilyName`, or `Suffix` attributes is required for object update.",
       optional: true,
     },
-    middle_name: {
+    middleName: {
+      label: "Middle Name",
       type: "string",
       description: "Middle name of the person. The person can have zero or more middle names. The `DisplayName` attribute or at least one of `Title`, `GivenName`, `MiddleName`, `FamilyName`, or `Suffix` attributes is required for object update.",
       optional: true,
     },
-    family_name: {
+    familyName: {
```

Comment:
Can we move `familyName` to the app file? It is used here and in `create-customer`